### DISCUSSION TITLE
Question about "flashing" sd-card on linux

### DISCUSSION THREAD

**Discussion by birdspider** [eduardofilo/RG350_adam_image#101] [General]
About the Installation Documentation:

> Fresh installation
> 1. Download the dump file from the releases section of this repository.
> 2. Flash the file (img.xz) without decompressing with Balena Etcher on a microSD card of at least 4GB.

is flashing the equivalent of `dd if= of=` ? or do I need to do something fancy on linux ?

EDIT: nvm, just saw that is Balena Etcher is also available on linux (still interested if it does something else than `dd`)

**Comment by eduardofilo** [comment]
Yes, it is equivalent. 

With xz compressed images you can do (adapting the device of course):

```
xzcat adam_v1.2.img.xz | sudo dd of=/dev/mmcblk0 bs=2M status=progress conv=fsync
```
